dashboard/app/helpers/application_helper.rb

```diff
@@ -63,14 +63,11 @@ def activity_css_class(result)
   end
 
   def level_info(user, script_level, user_levels)
-    result = nil
     if user
-      ul = user_levels[script_level.level_id]
-      result = ul.try(:best_result) if ul
+      user_levels[script_level.level_id].try(:best_result) || 0
     else
-      result = client_state.level_progress(script_level)
+      0
```

Comment:
This used to return a css class but now returns an integer, it looks like.  Seems better not to redefine the signature of this public method if we don't have to.

Could you you document this method to explain what it is expected to return?